packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -79,6 +85,10 @@ const AppModals = ({
         ComponentToLoad = <CloseUKAccountModal />;
     }
 
+    if (is_logged_in && client_country === 'br' && !has_read_scam_message) {
```

Comment:
Move this condition to `ui-store.js` the same way as other modals.
You can call it `is_warning_scam_message_modal_visible`